package.json

```diff
@@ -33,7 +33,9 @@
 		"test": "echo \"Error: no test specified\" && exit 1"
 	},
 	"dependencies": {
-		"html-webpack-plugin": "^5.5.4",
+		"archimate-js": "^0.0.4",
+		"bpmn-js": "^17.8.1",
+		"dmn-js": "^16.4.0",
```

Comment:
this dependency seems to be unused

Reply:
Same, is for another feature can be removed here.